OmiseSwift/Payment.swift

```diff
@@ -0,0 +1,58 @@
+import Foundation
+
+
+public enum Payment {
+    case card(Card)
+    case offsite(OffsitePayment)
+}
+
+extension Payment: Equatable {
+    /// Returns a Boolean value indicating whether two values are equal.
+    ///
+    /// Equality is the inverse of inequality. For any values `a` and `b`,
+    /// `a == b` implies that `a != b` is `false`.
+    ///
+    /// - Parameters:
+    ///   - lhs: A value to compare.
+    ///   - rhs: Another value to compare.
+    public static func ==(lhs: Payment, rhs: Payment) -> Bool {
```

Comment:
Is it possible to do avoid doing this by having `Payment`, `OffsitePayment` and `Card` implements `==` ?